scripts/migrations/04-apply-keywords.py

```
@@ -0,0 +1,73 @@
+Strategy: text-based insertion (not YAML round-trip) to preserve formatting
+of description/compiled fields. Insert `keywords:` block just before the
+closing `---` of frontmatter.
+"""
+from __future__ import annotations
+
+import re
+import sys
+import yaml
+from pathlib import Path
+
+sys.path.insert(0, "/tmp")
```

Ruff (0.15.13)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)